docs/3.0/tutorials/scraping.mdx

```diff
@@ -0,0 +1,503 @@
+
+```python
+@flow(log_prints=True)
+def analyze_repo_health(repos: [str]):
+    """Analyze issue health metrics for GitHub repositories"""
+    all_issues = []
+
+    for repo in repos:
+        for page in range(1, 3):  # Get first 2 pages
+            issues = fetch_page_of_issues(repo, page)
+            if not issues:
+                break
+            all_issues.extend(issues)
+    
+    # Run issue analysis tasks in parallel
+    for issue in all_issues:
+        analyze_issue.submit(issue)
+    
+    # Wait for all analysis tasks to complete
+    for detail in issue_details:
+        result = detail.result()
+        print(f"Analyzed issue #{result['number']}")
+```
+
+<Expandable title="full example">
+Run the following code to see pagination and parallel tasks in action:
+
+```python
+from typing import List, Optional
+import httpx
+
+from prefect import flow, task
+
+
+@flow(log_prints=True)
+def analyze_repo_health(repos: List[str]):
+    """Analyze issue health metrics for GitHub repositories"""
+    for repo in repos:
+        print(f"Analyzing {repo}...")
+        
+        # Fetch and analyze all issues
+        fetch_repo_issues(repo)
+
+
+@flow
+def fetch_repo_issues(repo: str):
+    """Fetch all issues for a single repository"""
+    all_issues = []
+    page = 1
+    
+    for page in range(1, 3):  # Limit to 2 pages to avoid hitting rate limits
+        issues = fetch_page_of_issues(repo, page)
+        if not issues or len(issues) == 0:
+            break
+        all_issues.extend(issues)
+        page += 1
+
+    issue_details = []
+    for issue in all_issues[:5]:  # Limit to 5 issues to avoid hitting rate limits
+        issue_details.append(
+            fetch_issue_details.submit(repo, issue['number'])  # Make requests in parallel
+        )
+    
+    details = []
+    for issue in issue_details:
+        details.append(issue.result())
+
+    return details
+
+
+@task(log_prints=True)
+def fetch_page_of_issues(repo: str, page: int = 1) -> Optional[dict]:
+    """Fetch a page of issues for a GitHub repository"""
+    try:
+        response = httpx.get(
+            f"https://api.github.com/repos/{repo}/issues",
+            params={"page": page, "state": "all", "per_page": 100}
+        )
+        response.raise_for_status()
+        return response.json()
+    except Exception as e:
+        print(f"Error fetching issues for {repo}: {e}")
+        return None
+
+
+@task
+def fetch_issue_details(repo: str, issue_number: int) -> dict:
+    """Fetch detailed information about a specific issue"""
+    response = httpx.get(f"https://api.github.com/repos/{repo}/issues/{issue_number}")
+    issue_data = response.json()
+    
+    return issue_data
+
+
+if __name__ == "__main__":
+    analyze_repo_health([
+        "PrefectHQ/prefect",
+        "tensorflow/tensorflow",
+        "huggingface/transformers"
+    ])
+```
+</Expandable>
+
+## Structure your code with dependent subflows and tasks
```

Comment:
```suggestion
## Structure your code with dependent nested flows and tasks
```

Esewhere in the docs we refer to them as nested flows, good to keep the consistency.